tests/caasp/stack_reboot.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Reboot cluster if update test isn't scheduled
+# Maintainer: Martin Kravec <mkravec@suse.com>
+
+use parent 'caasp_controller';
+use caasp_controller;
+
+use strict;
+use testapi;
+use caasp 'script_retry';
+use lockapi 'mutex_create';
+
+sub run {
+    switch_to 'xterm';
+    my $admin = 'admin.openqa.test';
+
+    record_info "Reboot admin";
+    script_run "ssh $admin 'reboot'", 0;
+    # Wait until reboot
+    script_retry "ping -c1 -W1 $admin", expect => 1, retry => 3, delay => 10;
+    # Wait until velum starts
+    script_retry "curl -kLI -m5 $admin | grep _velum_session";
+
+    record_info "Reboot cluster";
+    assert_script_run "ssh $admin './update.sh -r' | tee /dev/$serialdev | grep EXIT_OK";
+    # Wait until reboot
+    script_retry 'kubectl get nodes', expect => 1, retry => 3, delay => 10;
+    # Wait until kubernetes starts
+    script_retry 'kubectl get nodes';
```

Comment:
You know what you could do because you have a comment above each command: pass instead a string to the function as the fail message, as we have for assert_script_run

Reply:
I replaced comments with record_info for better output readability.
Did not want to use fail_message because script_retry calls can get quite long:
```
script_retry "curl -kLI -m5 $admin | grep _velum_session", expect => 1, retry => 3, delay => 10, fail_message => 'Wait until admin node powers off';
```
What do you think?